src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx

```diff
@@ -803,7 +822,7 @@ export class SavedObjectsTable extends Component<SavedObjectsTableProps, SavedOb
     ];
 
     const availableNamespaces = namespaceRegistry.getAll() || [];
-    if (availableNamespaces && availableNamespaces.length > 0) {
+    if (availableNamespaces?.length) {
```

Comment:
Based on line 824, `availableNamespaces` is always an array:
```suggestion
    if (availableNamespaces.length) {
```